core/org.openjdk.jmc.flightrecorder/src/main/java/org/openjdk/jmc/flightrecorder/jdk/JdkAttributes.java

```diff
@@ -579,6 +579,8 @@ public <U> IMemberAccessor<IQuantity, U> customAccessor(IType<U> type) {
 						return JdkAttributes.TLAB_SIZE.getAccessor(type);
 					} else if (type.getIdentifier().equals(JdkTypeIDs.ALLOC_OUTSIDE_TLAB)) {
 						return JdkAttributes.ALLOCATION_SIZE.getAccessor(type);
+					} else if (type.getIdentifier().equals(JdkTypeIDs.OBJ_ALLOC_SAMPLE)) {
+						return JdkAttributes.SAMPLE_WEIGHT.getAccessor(type);
```

Comment:
I'm going to revert this piece and branch on whether we expect ObjectAllocationSample events -- I initially audited uses within `core`, however I missed some places in the UI that would likely have slightly different behavior.
I suspect it would be generally helpful to include sample weight in this attribute, but if we make that change, I'd prefer to do it in isolation from this enhancement.